test/TorchSharpTest/TestTorchTensorBugs.cs

```diff
@@ -531,8 +531,8 @@ public Module510(int in_channels, int out_channels, int kernel_size = 3, int str
                     ReLU(inplace: true)
                 );
 
-                temp.weight = Parameter(torch.randn(temp.weight.shape));
-                temp.bias = Parameter(torch.randn(temp.bias.shape));
+                temp.weight = Parameter(torch.randn(temp.weight!.shape));
```

Comment:
Can you actually null `weight` of `BatchNorm1d`? Is it a legal operation that does not break things?

Reply:
PyTorch allows it. Not sure what happens after...